sql/src/lib.rs

```diff
@@ -136,11 +140,16 @@ impl QueryBuilder for SQLiteQueryBuilder {
             &Boolean(v) => self.push_sql(if v { "1" } else { "0" }),
             &Long(v) => self.push_sql(v.to_string().as_str()),
             &Double(OrderedFloat(v)) => self.push_sql(v.to_string().as_str()),
-
-            // These are both `Rc`. We can just clone an `Rc<String>`, but we
-            // must make a new single `String`, wrapped in an `Rc`, for keywords.
-            &String(ref s) => self.push_static_arg(s.clone()),
-            &Keyword(ref s) => self.push_static_arg(Rc::new(s.as_ref().to_string())),
+            // These are both `Rc`. Unfortunately, we can't use that fact when
+            // turning these into rusqlite Values.
+            &String(ref s) => {
+                let v = Rc::new(rusqlite::types::Value::Text(s.as_ref().clone()));
```

Comment:
But this will destroy the value of interning strings in the query parser, right?  Not that we intern yet; once we do, we should be able to coalesce repeated values and avoid cloning the same string more than once.  I guess that'll be good enough.